config_spec.toml

```diff
@@ -0,0 +1,19 @@
+[[param]]
+name = "address"
+type = "String"
+optional = false
+doc = "LND GRPC address, kindly note that the address must start with https:// ."
+
+
+[[param]]
+name = "cert"
+type = "String"
+optional = false
+doc = "The path to LND tls certificate file"
```

Comment:
While we're here, will you make a note int the docs for cert + macaroon that we currently require abosluote paths.